APMrover2/Log.cpp

```diff
@@ -284,11 +311,11 @@ void Rover::Log_Write_Attitude()
     DataFlash.Log_Write_Attitude(ahrs, targets);
 
 #if AP_AHRS_NAVEKF_AVAILABLE
- #if defined(OPTFLOW) and (OPTFLOW == ENABLED)
+#if defined(OPTFLOW) and (OPTFLOW == ENABLED)
```

Comment:
On Mon, 29 Aug 2016, Lucas De Marchi wrote:

> > @@ -284,11 +311,11 @@ void Rover::Log_Write_Attitude()
> >      DataFlash.Log_Write_Attitude(ahrs, targets);
> > 
> >  #if AP_AHRS_NAVEKF_AVAILABLE
> > - #if defined(OPTFLOW) and (OPTFLOW == ENABLED)
> > +#if defined(OPTFLOW) and (OPTFLOW == ENABLED)
>
>       How is this more readable @lucasdemarchi (because you were the
>       one saying you liked it this way)?
> 
> Maybe because I'm used to it.

I'd prefer a simple "#if OPTFLOW == ENABLED" to take proper advantage of 
-Wundef....